out/notebooks/spl2/installer.ts

```diff
@@ -152,16 +162,18 @@ export async function installMissingSpl2Requirements(context: ExtensionContext,
  *          meets out minimum Java major version
  */
 function isJavaVersionCompatible(javaLoc: string): boolean {
+    let output;
     try {
         const javaVerCmd = child_process.spawnSync(javaLoc, ['-version'], { encoding : 'utf8' });
         if (!javaVerCmd || javaVerCmd.stdout) {
             return false;
         }
         // java -version actually writes to stderr so check for a match there
-        const match = javaVerCmd.stderr.toString().match(/version \"([0-9]+)\.[0-9]+\.[0-9]\"/m);
+        output = javaVerCmd.stderr.toString();
+        const match = output.match(/version \"([0-9]+)\.[0-9]+\.[0-9].*\"/m);
```

Comment:
This fixes a linux issue where Java version of format `openjdk version "17.0.8.1"` was not being parsed correctly